packages/babel-preset-env/src/index.ts

```diff
@@ -354,23 +351,20 @@ option \`forceAllTransforms: true\` instead.
   const exclude = transformIncludesAndExcludes(optionsExclude);
 
   const compatData = getPluginList(shippedProposals, bugfixes);
-  const shouldSkipExportNamespaceFrom =
-    (modules === "auto" && api.caller?.(supportsExportNamespaceFrom)) ||
-    (modules === false &&
-      !isRequired("transform-export-namespace-from", transformTargets, {
-        compatData,
-        includes: include.plugins,
-        excludes: exclude.plugins,
-      }));
+  const modules =
+    optionsModules === "auto"
+      ? api.caller(supportsStaticESM)
```

Comment:
`?.` removed from `api.caller` because every Babel 7 version provides api.caller. This was only needed during the beta period.